airbyte-integrations/connectors/source-mssql/build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.23.15'
+    cdkVersionRequired = '0.23.17'
```

Comment:
Any reason to bump up the cdk version? or just bumping to latest?

Reply:
yes - bumping to the latest so the new iterator interface introduced a few PRs ago can be used